ollama/_utils.py

```diff
@@ -15,12 +15,12 @@ def _parse_docstring(doc_string: Union[str, None]) -> dict[str, str]:
   if not doc_string:
     return parsed_docstring
 
-  key = hash(doc_string)
+  key = str(hash(doc_string))
```

Comment:
`mypy` complains about a type mismatch for `key` since `hash(...)` returns an `int` but may be set to a `'args'`, i.e. string, later